Repositories.md

```diff
@@ -0,0 +1,78 @@
+# Guidelines For Repository Inclusion
+
+This document outlines the requirements and guidelines for a Helm repository to
+be listed in the distributed search hosted by the Helm Project. This document
+may be updated at any time.
+
+## Process To Add A Repository
+
+There is a two step process for a repository to be added to the distributed
+search.
+
+1. Provide the repositories contact information to the Helm maintainers by filling
+out the form at ABC (link to be inserted). This contact information is when there is a need for the Helm
+project to contact those who managed a repository (e.g., it goes offline). If the
+repository is owned by a team or company we suggest using a mailing list.
+2. Create a pull request against this repository adding the repository location
+in the file at ABC (link to be inserted).
+
+Once both of these are complete the repository and its contents will be listed
+in the distributed search.
+
+## Repository Best Practices
+
+Repositories listed in the distributed search should be of a high quality and
```

Comment:
How will we verify this?

Are there any hard constraints that we can put in place? Should we?

Reply:
I should take out "high quality" as that's not checkable. At first we will likely give it the eye check. When we have more time we can write automation to help folk with that.